docs/get-started/product-logo-library.njk

```diff
@@ -0,0 +1,136 @@
+
+  <div class="multi-column--min-400-wide">
+    <div>
+      <h4>Standard product logo</h4>
+      {% call getstarted.example(palette='lightest') %}
+        <img src="{{ '/assets/get-started/product-logo-library/product-logo-standard.png' | url }}" alt="Standard product logo" style="--example-img-max-width: 266px;">
+      {%- endcall %}
+    </div>
+    <div>
+      <h4>Product logo with variant</h4>
+      {% call getstarted.example(palette='lightest') %}
+        <img src="{{ '/assets/get-started/product-logo-library/product-logo-variant.png' | url }}" alt="Product logo with variant" style="--example-img-max-width: 268px;">
+      {%- endcall %}
+    </div>
+  </div>
+
+{%- endcall %}
+
+{% call getstarted.section("Product logo colors") -%}
+
+  <p>Red Hat product logos are produced in two full-color versions: standard and reverse. They are also produced in two one-color versions: white and black.</p>
+
+  <img src="{{ '/assets/get-started/product-logo-library/product-logo-colors.png' | url }}" alt="Product logo colors" style="--example-img-max-width: 1000px;">
+
+{%- endcall %}
+
+{% call getstarted.section("Product logo sizing") -%}
+
+  <p>Product logo names with different line counts will appear larger or smaller when they are the same height. To fix this, adjust the height of each product logo manually so the fedora icons are all the same height.</p>
+
+  {% call getstarted.example(palette='lightest') %}
+    <img src="{{ '/assets/get-started/product-logo-library/product-logo-sizing.png' | url }}" alt="Product logo sizing" style="--example-img-max-width: 835px;">
+  {%- endcall %}
+
+{%- endcall %}
+
+{% call getstarted.section("Best practices") -%}
+
+  <h4>Inconsistent sizing</h4>
+
+  <p>Do not assign product logos the same height value, adjust the height of each product logo manually so the fedora icons are all the same height.</p>
+
+  {% call getstarted.example(palette='wrong2') %}
+    <img src="{{ '/assets/get-started/product-logo-library/best-practice-1.png' | url }}" alt="Product logos best practice 1" style="--example-img-max-width: 793px;">
+  {%- endcall %}
+
+  <h4>One-color product logos</h4>
+
+  <p>Do not use the one-color version of any product logo or the Red Hat logo on any web property.</p>
+
+  {% call getstarted.example(palette='wrong2') %}
+    <img src="{{ '/assets/get-started/product-logo-library/best-practice-2.png' | url }}" alt="Product logos best practice 2" style="--example-img-max-width: 503px;">
+  {%- endcall %}
+
+{%- endcall %}
+
+{% call getstarted.section("Learn more") -%}
+
+  <p>To learn more about what assets are available in each library and how to find them in your Adobe CC apps, visit <a href="https://docs.google.com/presentation/d/11iqtI1ew6lAiUHXHY8u7nxUWOppg0n9W51Kj1CCFhBA/edit?usp=sharing&urp=gmail_link">this page</a>.</p>
```

Comment:
Reword to `To learn more about what assets are available in each library and how to find them in your Adobe CC apps, view <a href="https://docs.google.com/presentation/d/11iqtI1ew6lAiUHXHY8u7nxUWOppg0n9W51Kj1CCFhBA/edit?usp=sharing&urp=gmail_link">these instructions</a>.`